packages/x-charts/src/BarChart/BarPlot.tsx

```diff
@@ -81,7 +81,7 @@ function BarPlot(props: BarPlotProps) {
   const classes = useUtilityClasses();
 
   return (
-    <BarPlotRoot className={classes.root}>
+    <BarPlotRoot data-drawing-container className={classes.root}>
```

Comment:
Should this be set on the parent? Otherwise when using composition `data-drawing-container` will always be included, even if you don't want it.

Reply:
I'm not sure, but couldn't this cause a breaking change if someone isn't expecting this behavior to change? From now on, everything inside `BarPlot`, `LinePlot` and `AreaPlot` will be considered inside the drawing area, right?

Also, it's different from the pattern set for `MarkPlot` and `ScatterPlot`.